pkg/kube/secret/secret_builder.go

```diff
@@ -51,7 +51,7 @@ func (b *builder) SetByteData(stringData map[string][]byte) *builder {
 	b.data = newStringDataBytes
 	return b
 }
-func (b *builder) SetStringData(stringData map[string]string) *builder {
+func (b *builder) SetStringMapToData(stringData map[string]string) *builder {
```

Comment:
That name should make it more clear that we save into `secret.Data` not `secret.StringData`